daisy/workflow/step_create_instances.go

```diff
@@ -108,14 +108,14 @@ func (c *CreateInstances) run(w *Workflow) error {
 				default:
 					startup = "startup-script-url"
 				}
-				inst.AddMetadata(map[string]string{startup: path.Join(w.sourcesPath, ci.StartupScript)})
+				inst.AddMetadata(map[string]string{startup: "gs://" + path.Join(w.bucket, w.sourcesPath, ci.StartupScript)})
```

Comment:
I set that variable in the preceding switch as windows and linux are slightly different.